src/main/java/de/hhu/abschlussprojektverleihplattform/controllers/ConflictResolutionController.java

```diff
@@ -2,18 +2,27 @@
 package de.hhu.abschlussprojektverleihplattform.controllers;
 
 import de.hhu.abschlussprojektverleihplattform.model.LendingEntity;
+import de.hhu.abschlussprojektverleihplattform.model.UserEntity;
+import de.hhu.abschlussprojektverleihplattform.service.LendingService;
+import de.hhu.abschlussprojektverleihplattform.service.ProductService;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
+
 import java.util.List;
 
 //@Controller
 public class ConflictResolutionController {
 
+    @Autowired
+    private LendingService lendingService;
+    @Autowired
+    private ProductService productService;
+
```

Comment:
besser keine fieldinjection

Reply:
Ist momentan mal so mal so, das muss in allen Controller noch vereinheitlicht werden